straxen/plugins/acqmon_processing.py

```diff
@@ -296,7 +307,8 @@ def handle_starts_and_stops_outside_of_run(
 
     @staticmethod
     def fake_hit(start, dt=1, length=1):
-        hit = np.zeros(1, strax.hit_dtype)
+        aqmon_hit_dtype = strax.unpack_dtype(self.deps['aqmon_hits'].dtype_for('aqmon_hits'))
+        hit = np.zeros(1, aqmon_hit_dtype)
```

Comment:
I think you need to add self and remove the static method

Reply:
True, thanks